remix.init/entry.server.tsx

```diff
@@ -0,0 +1,31 @@
+import type { EntryContext } from "@remix-run/server-runtime";
```

Comment:
Made the change after chatting with the Remix team. See https://github.com/netlify/remix-compute/pull/51 for more context.